website/players.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Carbon\Carbon;
+use Cocur\HumanDate\HumanDate;
+
+$config = require '/var/creds/creds.php';
+$db = $config['database'];
+
+$conn = mysqli_connect($db['host'], $db['user'], $db['password'], $db['dbname']);
+if (!$conn) {
+    die('Connection failed: ' . mysqli_connect_error());
+}
+
+$offset = (int)$_GET['offset'];
+
```

Comment:
I think we should try to be consistent with the game. It does its order by as this:

`ORDER BY rating DESC, (wins - losses) DESC, wins DESC`